databases/iplike/Portfile

```diff
@@ -14,20 +15,19 @@ long_description    C implementation of OpenNMS's plpgsql iplike function, \
                     preferred as it offers better performance.  Allows for \
                     complex IP address queries on a text field.
 
-homepage            http://www.opennms.org/wiki/IPLIKE
+depends_lib         port:postgresql16
 
-master_sites        sourceforge:opennms
-
-depends_lib         port:postgresql90
-
-depends_run         port:postgresql90-server
```

Comment:
is there no dependency on the `postregsql16-server` needed anymore?

Reply:
It was never needed, the -server portfile just has configuration to start a server in launchd, it has none of the postgresql code packaged.